phy/cluster/algorithms.py

```diff
@@ -155,7 +164,7 @@ def __init__(self, tempdir=None, probe=None, **kwargs):
     def _create_filter(self):
         rate = self._kwargs['sample_rate']
         low = self._kwargs['filter_low']
```

Comment:
that's always been the case. I assume these parameters do not normally depend on the sample rate, but for the high frequency, you want to set the high one to the max, i.e. about half of the sample rate or something... @shabnamkadir ?